01_fib/nicholascross/main_test.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"bytes"
+	"strconv"
+	"testing"
+)
+
+func TestMainOutput(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+
+	main()
+
+	expected := strconv.Quote("1 1 2 3 5 8 13")
+	actual := strconv.Quote(buf.String())
+
+	if expected != actual {
+		t.Errorf("Unexpected output in main()")
+	}
+}
+
+func TestFib(t *testing.T) {
+	expectFib(7, "1 1 2 3 5 8 13", t)
```

Comment:
Since you are testing over a range of different cases, you can do something similar to what is shown here [Testing Range Cases](https://goplay.space/#Zy2ta-z8vrO)

```go
func TestIsEven(t *testing.T) {
	cases := []struct {
		length   int
		sequence string
	}{
		{length: 7, sequence: "1 1 2 3 5 8 13"},
		{length: -7, sequence: "1 -1 2 -3 5 -8 13"},
		{length: 0, sequence: ""},
		{length: 1, sequence: "1"},
		{length: 10, sequence: "1 1 2 3 5 8 13 21 34 55"},
	}

	for _, c := range cases {
		expectFib(c.length, c.sequence, t)
	}
}
```